components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html

```diff
@@ -0,0 +1,104 @@
+{{ $data_path := index $.Site.Data (split (.Get "data_path") "/") }}
+{{ range $data_path }}
+{{ $id := urlize .name }}
+{{ $command := strings.TrimPrefix "chef-automate " .name }}
+{{ if or (not .compatible_with) (eq .compatible_with "AutomateHA") }}
+<h3 id="{{ $id }}">
+   {{ .name }}</h2>
+   <p>{{ .description }}</p>
+   <h4 id="{{ $id }}-usage">
+   Usage
+</h3>
+<pre><code>{{ .usage }}</code></pre>
+<h4 id="{{ $id }}-flags">
+Flags</h3>
+<div class="fake-code grid-x">
+   <table>
+      <tr class="table-heading">
+         <th>Flags</th>
+         <th>Description</th>
+      </tr>
+      {{- range .options -}}
+      <tr>
+         {{ if or (not .compatible_with_options) (eq .compatible_with_options "AutomateHA") }}
+         <td>
+            <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
+         </td>
+         <td>
+            <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
+         </td>
+         {{ end }}
+      </tr>
+      {{- end -}}    
+      {{- range .inherited_options -}}
+      <tr>
+         {{ if or (not .compatible_with_options) (eq .compatible_with_options "AutomateHA") }}
+         <td>
+            <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
+         </td>
+         <td>
+            <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
+         </td>
+         {{ end }}
+      </tr>
+      {{- end -}}
+   </table>
+</div>
+<h4>Supported On</h4>
+<p>{{ .supported_on }}</p>
```

Comment:
Something like this:
```suggestion
<p>This command is supported on the following node types: {{ .supported_on }}</p>
```

Reply:
You should probably add something where, if there aren't any types listed, it says "This command is supported on all node types."

Reply:
On the HA page, all the commands will be supported with one node. 
The UX team and the leads have decided on the header with the content.